Components/badge.css

```diff
@@ -0,0 +1,55 @@
+/* BADGE COMPONENT */
+
+/* Badge on Avatar */
+
+.badge-on-avatar {
```

Comment:
suggest you move these lines below, as you don't have badge in the top and directly talking about the avatar and badge

Reply:
Oh, right! Makes sense.